event_manager/settings.py

```diff
@@ -151,5 +152,24 @@
     'SIGNING_KEY': 'thisismysecret'
 }
 
+MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
+
+if not DEBUG:
+    STATICFILES_DIRS += [
+        os.path.join(BASE_DIR, 'media'),
+    ]
+
+    AWS_ACCESS_KEY_ID = os.environ.get('AWS_ACCESS_KEY_ID')
+    AWS_SECRET_ACCESS_KEY = os.environ.get('AWS_SECRET_ACCESS_KEY')
+    AWS_STORAGE_BUCKET_NAME = os.environ.get('AWS_STORAGE_BUCKET_NAME')
```

Comment:
@usmandap Bucket name can be configured into source control. i can't see any security issues with that

Reply:
@abdulmoizeng You mean we don't need to mention bucket name using environments?

Reply:
@abdulmoizeng I think all of it can be kept in the same place (i..e all system level configs)

Reply:
@mashhoodr normally bucket isn't kept in system level configs. I presume following rule to  keep things in system level.

**Any thing which is a security concern being in source control should be kept in system level**

Reply:
I dont think thats a valid rule, you can keep other configs on system level including which kind of ENV you are in etc.